stable/admin/templates/_helpers.tpl

```diff
@@ -265,4 +265,37 @@ Renders the admin service name for external access based on the service type
   {{- else -}}
 {{ .Values.admin.domain }}
   {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+Renders the annotations for the LoadBalancer admin service
+*/}}
+{{- define "admin.externalAccessAnnotations" -}}
+  {{- if eq (default "LoadBalancer" .Values.admin.externalAccess.type) "LoadBalancer" }}
+    {{- if .Values.admin.externalAccess.annotations }}
+{{ toYaml .Values.admin.externalAccess.annotations | trim }}
+    {{- else -}}
+      {{- if .Values.cloud.provider }}
+        {{- if eq .Values.cloud.provider "amazon" }}
+          {{- if .Values.admin.externalAccess.internalIP }}
+service.beta.kubernetes.io/aws-load-balancer-internal: "true"
+service.beta.kubernetes.io/aws-load-balancer-scheme: "internal"
+          {{- else }}
+service.beta.kubernetes.io/aws-load-balancer-type: "external"
+service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: "ip"
```

Comment:
This will work with the [Amazon EKS VPC CNI plugin](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) only. For alternative CNI plugins, it should be set to `instance`. For example:

```yaml
admin:
  externalAccess:
    enabled: true
    annotations:
      "service.beta.kubernetes.io/aws-load-balancer-type": "external"
      "service.beta.kubernetes.io/aws-load-balancer-nlb-target-type": "instance"
      "service.beta.kubernetes.io/aws-load-balancer-scheme": "internet-facing"
```